tests/lib/bash/test_header

```diff
@@ -332,7 +332,7 @@ grep_ok() {
     OPTS="$*"
     local TEST_NAME
     TEST_NAME="$(basename "${FILE}")-grep-ok"
-    if grep "${OPTS}" -q -e "${BRE}" "${FILE}"; then
+    if grep ${OPTS} -q -e "${BRE}" "${FILE}"; then
```

Comment:
Is this okay?

Reply:
Or does it have to be an array instead like https://github.com/koalaman/shellcheck/wiki/SC2086#exceptions

Reply:
This fails shellcheck (which prompted the incorrect quoting in the first place :grimacing: )

Reply:
By experiment, the array approach works for options and no options, I'll suggest the change in two places...